en/gps_compass/rtk_gps.md

```diff
@@ -139,7 +154,33 @@ See the [EKF2 GPS Configuration](../advanced_config/tuning_the_ecl_ekf.md#gps) s
 - something that shows positioning of base, connection of RTK rover, survey in process. Some sort of short precision survey.
 -->
 
-
-## Vehicle Setup Example
+### Vehicle Setup Example
 
 The airframe build topic [DJI Flamewheel 450 with distance sensor and RTK GPS](../frames_multicopter/dji_flamewheel_450.md) describes an airframe setup with the Here+ RTK GPS and a Pixhawk 3 Pro.
+
+
+
+## RTK GPS that Output Yaw
+
+Some RTK GPS units have multiple antennas, and can output a yaw angle.
```

Comment:
I'd keep this a bit more generic, as RTK is not a requirement
```suggestion
Some GPS units have multiple antennas, and can output a yaw angle.
```

Reply:
> I'd keep this a bit more generic, as RTK is not a requirement

Are there any other units we know support this or that might do so in future? Or to put another way, should this move to the GPS page as "GPS that Output Yaw"?

A

Reply:
I'd move it as I don't think I'd expect it here.
There's this for example: https://www.unmannedsystemstechnology.com/2016/03/vectornav-technologies-introduces-vn-360-gps-compass-heading-position-sensor/ (no PX4 support though)